main_page.php

```diff
@@ -2,6 +2,23 @@
     require_once('functions.php');
     $db = createDBConnection();
     $recipes = fetchAllRecipes($db);
+    if (isset($_POST['edit'])) {
```

Comment:
maybe better to have this in a callable function?

Reply:
Maybe although my thoughts for not including it in a function was because I don't plan on using it again?